### PR TITLE
Add Xode Project

### PR DESCRIPTION
**Pull request by hgminerva**
Xode Parachain Project (3344)

### CODE REVIEW

File: packages/apps-config/src/endpoints/productionRelayKusama.ts
Comment (JelliedOwl) [inline packages/apps-config/src/endpoints/productionRelayKusama.ts:1]: I think the CI process is very picky about alphabetical order, and this (L8) will fail the check when it gets run by the maintainer.

Comment (hgminerva) [reply packages/apps-config/src/endpoints/productionRelayKusama.ts:1]: Got it, thank you so much, updating now

**Comment by TarikGul** [comment]
Conflicts need to be resolved before we can merge.

**Comment by TarikGul** [comment]
Apologies again, looks like there are conflicts. Once fixed i will prioritise this PR.

**Comment by hgminerva** [comment]
Done fixing, thanks so much.

**Review by TarikGul — APPROVED** [review]

**Pull request merged**
